### ISSUE TITLE
Demos/benchmarks/chaos.py has broken

### ISSUE THREAD

**Issue by fenrus75** [cython/cython#1962]
```
Error compiling Cython file:
------------------------------------------------------------
...
    def dist(self, other):
        return sqrt((self.x - other.x) ** 2 +
                    (self.y - other.y) ** 2 +
                    (self.z - other.z) ** 2)

    @cython.locals(self="GVector", other="GVector")
                       ^
------------------------------------------------------------

chaos.py:36:24: Not a type
```

**Comment by scoder** [comment]
Thanks. Fixed in eb270f7e763f6f2313da447addfb4a29425d8672.

**Comment by fenrus75** [comment]
Thank you for the VERY quick fix!
